plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -36,8 +38,25 @@ type Query struct {
 // MapQuery type
 type MapQuery map[string]Query
 
+// HealthMetric struct tracking the number of attempted vs successful connections for each connection string
+type HealthMetric struct {
+	AttemptedQueries  int
+	SuccessfulQueries int
+}
+
 const defaultServer = "Server=.;app name=telegraf;log=1;"
 
+const typeAzureSQLDB = "AzureSQLDB"
```

Comment:
nit: same as earlier -it can be grouped under single const variable for readability

Reply:
Done. Created multiple groups to clarify which constants are related.